classify.py

```diff
@@ -339,22 +335,29 @@ def evaluate_samesource(desc, dataset, n_frequent_words, tokens_per_sample, max_
     acc = np.round(np.mean((lrs > 1) == y_all), 3)
     recall = np.round(np.mean(lrs[y_all == 1] > 1), 3)
     precision = np.round(np.mean(y_all[lrs > 1] == 1), 3)
-    Results = collections.namedtuple('Results', ['cllr', 'cllrmin', 'cllrcal', 'accuracy', 'recall', 'precision', 'lrs',
-                                                 'label'])
+    tnr = np.round(np.mean(lrs[y_all == 0] < 1), 3)
```

Comment:
why round? it is good practice to round as late as possible, for example when printing something on screen